contracts/toCKB-typescript/src/utils/config.rs

```diff
@@ -1,4 +1,12 @@
 pub const PLEDGE: u64 = 10000;
+pub const XT_CELL_CAPACITY: u64 = 100;
```

Comment:
the minimum size of sUDT is 130 bytes, we could set it 200.

```suggestion
pub const XT_CELL_CAPACITY: u64 = 200;
```